### DISCUSSION TITLE
Loading saved courses?

### DISCUSSION THREAD

**Discussion by theRealPDXMike** [Courseplay/Courseplay_FS22#665] [Q&A] [answered]
Is there a way to load a saved CP course on a different piece of equipment?  I seem to remember CP for FS19 you could do this, but either it isn't currently implemented or I'm going about it all wrong. What I'm trying to do is harvest a field with a combine leaving a straw swath, then come back later with a forage wagon or a baler to pick up the straw. Since the width is the same the second machine should be able to just load the same course as the combine.

**Comment by Redster999** [comment]
I just did it. Had a saved course in No mans land for harvester. Loaded the course with a smaller cultivator and clicked the course in the mini hud. The tool settings had changed. This would be nice for the baler, but ofcourse for my smaller cultivator i needed to generate a new route. And it worked, also with the same starting point

**Comment by Tensuko** [comment, marked as answer]
Save course -> load course.
Just like normal :)

**Reply by theRealPDXMike** [reply]
Ok, got it working with the baler. It does not work with the forage wagon. My problem getting it to load earlier was that I didn't clear the current course before trying to load the new course.
The forage wagon will load the course, but won't start the worker.

**Reply by mario-fehr** [reply]
> Ok, got it working with the baler. It does not work with the forage wagon. My problem getting it to load earlier was that I didn't clear the current course before trying to load the new course.
> The forage wagon will load the course, but won't start the worker. 

Yes because forage wagon worker is not implemented yet
